tests/pkunit_test.py

```diff
@@ -78,6 +78,16 @@ def test_file_eq_ndiff():
         actual_path=d.join("actual_conformance.ndiff"),
         ndiff_epsilon=1e-9,
     )
+    pkunit.file_eq(
+        expect_path=d.join("expect_rel.ndiff"),
+        actual_path=d.join("actual_rel.ndiff"),
+        abs_ndiff_epsilon=1e-13,
+    )
+    with pkunit.pkexcept("diffs detected:"):
```

Comment:
This is documented as "rel", but it's implicit. Make it the expect_default.ndiff, and then test the case of passing a rel epsilon. Rename the "conformance_2" to match the other names: rel, default, rel2, whatever.